### PR TITLE
Fix/staked capacity during writeprice update

### PR DESCRIPTION
**Pull request by Jayashsatolia403**
## Tests
Tasks to complete before merging PR:
- [ ]  Ensure system tests are passing. If not [Run them manually](https://github.com/0chain/0chain/actions/workflows/system_tests.yml) to check for any regressions :clipboard:
- [ ]  Do any new system tests need added to test this change? do any existing system tests need updated? If so create a PR at [0chain/system_test](https://github.com/0chain/system_test)
- [ ]  Merge your system tests PR to master AFTER merging this PR

## Associated PRs (Link as appropriate):
- blobber:
- gosdk:
- system_test:
- zboxcli:
- zwalletcli:
- Other: ...

### CODE REVIEW

File: code/go/0chain.net/smartcontract/storagesc/blobber.go [hunk 1]
```diff
@@ -142,9 +142,9 @@ func (sc *StorageSmartContract) updateBlobber(
 		return fmt.Errorf("error calculating staked capacity: %v", err)
 	}
 
-	if blobber.Capacity < stakedCapacity {
-		return fmt.Errorf("write_price_change: staked capacity(%d) exceeding total_capacity(%d)",
-			stakedCapacity, blobber.Capacity)
+	if blobber.SavedData < stakedCapacity {
```
Comment (Kishan-Dhakan) [inline code/go/0chain.net/smartcontract/storagesc/blobber.go]: Used can be less than staked capacity

Comment (dabasov) [reply code/go/0chain.net/smartcontract/storagesc/blobber.go]: yeah, it should be `blobber.SavedData > stakedCapacity`

Comment (Jayashsatolia403) [reply code/go/0chain.net/smartcontract/storagesc/blobber.go]: Oh yes, I was checking if used capacity is not getting less than new staked capacity. Let me make the changes.

**Review by dabasov — APPROVED** [review]
lgtm

**Pull request merged**